src/controller.cpp

```diff
@@ -509,55 +504,30 @@ geometry_msgs::Twist Controller::update(double target_x_vel,
 
   // trackin_error here represents the error between tracked link and position on plan
   controller_state_.tracking_error_lat = current_tracking_err.y();
-  controller_state_.tracking_error_ang = controller_state_.error_ang.at<0>();
+  controller_state_.tracking_error_ang = controller_state_.error_ang.errors().at<0>();
```

Comment:
Here the raw errors are used, instead of the filtered ones. We should first filter the error, and then use that one in the P I D actions. This can be done in a separate PR

Reply:
Would you make an issue for that then, and approve this PR?